dev-db/mariadb-connector-odbc/mariadb-connector-odbc-3.1.6.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 2018-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils multilib-minimal
+
+DESCRIPTION="MariaDB Connector/ODBC"
+HOMEPAGE="https://downloads.mariadb.org/connector-odbc/"
+SRC_URI="https://downloads.mariadb.org/interstitial/connector-odbc-${PV}/${P}-ga-src.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0/3.0"
+KEYWORDS="~amd64 ~x86"
+IUSE="ssl gnutls"
```

Comment:
Please sort this list. Do you happen to know if it works with libressl?

Reply:
Done.  I do not know about libressl.